examples/examples/tower-load.rs

```diff
@@ -308,14 +308,16 @@ async fn load_gen(addr: SocketAddr) -> Result<(), Err> {
         .layer(request_span::layer(req_span))
         .timeout(Duration::from_millis(200))
         .service(Client::new());
-    let mut interval = time::interval(Duration::from_millis(50));
+    let interval = tokio::time::interval(Duration::from_millis(50));
+    let mut interval = tokio_stream::wrappers::IntervalStream::new(interval);
+
     while interval.next().await.is_some() {
```

Comment:
we don't actually need the `Stream` impl here, so this can just be
```suggestion

    loop {
        interval.tick().await;
```